cypress/integration/pages/liveRadio/amp.js

```diff
@@ -0,0 +1,138 @@
+import config from '../../../support/config/services';
+// import envConfig from '../../../support/config/envs';
+// import appConfig from '../../../../src/app/lib/config/services';
+import describeForEuOnly from '../../../support/describeForEuOnly';
+import { hasLiveRadioPage } from '../../../support/serviceHasPageType';
+import { getLiveRadioUrl } from '../../../support/getPageTypeUrl';
+
+const runTests = service => {
+  describe(`Live Radio - Amp - ${service}`, () => {
+    before(() => {
+      cy.visit(`${getLiveRadioUrl(service)}.amp`);
+    });
+
+    // will be addressed by https://github.com/bbc/simorgh/issues/2750
+    // it('Metadata', () => {
```

Comment:
Does cypress have a way to disable tests without the huge comment?  eg, `xdescribe`, `xit`

Reply:
I thought about doing that but thought commenting would be easier to see what is disabled and enable by uncommenting